automates/utils/astpp.py

```diff
@@ -0,0 +1,98 @@
+"""
```

Comment:
Right. I thought this file would eventually end up here so I added it twice. The reason it was in scripts originally is because I couldn't get it to work when it was in the 'utils' folder. I'll delete the one in 'utils' then.